### PR TITLE
feat: expand on export bot data docs

### PR DESCRIPTION
**Pull request by devguilhermy**
Adds code for getting the conversation messages as well when using the API
Adds other methods for sending the conversation from the bot - using the summary agent
Adds small updates

### CODE REVIEW

File: pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+[Hooks](../../studio/hooks) are functions that are executed under the hood every time there's a new message from the user or from the bot. You can create hooks that build the conversation history as it happens. They work similarly to the Summary Agent but with hooks you can customize how the history is built and there's no limitation for its size.
+
+#### Advantages
+
+This solution is useful if you need to have all details about the conversation. Using the Hooks for exporting conversations is great because the export happens during the conversation (or when it ends) so you don't need to make manual or scheduled requests to our API from external services.
+
+#### Disadvantages
+
+The disadvantage of using Hooks for exporting conversations is that the bot may get slow on conversations with a lot of
+messages or with long messages, because all of those messages will be stored in the chatbot memory. Another
```
Comment (ptrckbp) [inline pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx:37]: "may slow down a bit as there slight latency added to store a message."

it shouldn't be a big impact

File: pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Exporting Conversations from within your Bot
+
+Botpress Cloud provides you with several different ways to export a conversation to external services, each one has its use cases and advantages. Let's take a look at them.
+
+### Using the Summary Agent
+
+The [Summary Agent](../../studio/agents/#summary-agent) listens to all new messages in the conversation to build a summary and a transcript.
+
+- The summary is available at `conversation.SummaryAgent.summary` and contains an explanation of what happened in conversation, like "The user asked for the opening hours and the bot answered by saying the business works 24/7". You can choose how detailed the summary is by setting the max tokens in the Agent settings.
+- The transcript is available at `conversation.SummaryAgent.transcript` and contains the history of the conversation going back up to 100 turns. You can set the amount of turns in the Summary Agent settings.
```
Comment (ptrckbp) [inline pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx:12]: can we set the turn to 0 and have it fetch everything?

File: pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx (hunk 1 above)
Comment (ptrckbp) [reply pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx:37]: Actually, I think the summary agent method would be slower as this just stores the information, and the summary agent does an LLM summarization action.

**Review by ptrckbp — DISMISSED** [review]
Pretty good, I like how you added comments to the code! 

I have a few small requests (see the comments)

File: pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+#### Disadvantages
+
+The disadvantage of using the Summary Agent for exporting conversations is that after the amount of messages surpasses
+the max amount of turns for the transcript, you won't have the full conversation anymore. If you expect your users to
+have long conversations, use the Hooks method below. Another disadvantage is that the Agent only has access to the
+current session, so if the user is resuming a conversation, you won't have the previous messages. To get the whole
+conversation at any moment, use the [API method for exporting conversations](./with-the-api)
+
+### Using Studio Hooks
```
[the remark this answers is not in the document]
Comment (devguilhermy) [reply pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx:26]: What do you mean by outside? They are a Studio feature that allows you to read and write to external services

File: pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx (hunk 1 above)
Comment (devguilhermy) [reply pages/cloud/exporting-bot-data/exporting-conversations/from-the-bot.mdx:37]: Okay since we don't have actual metrics I won't mention speed

**Pull request merged**